Sql tables

WirelessWisdom opened the pull request.

Commit:
creation of tables

Commit:
crud rename

mestolin commented on db/tables/tasks.py:
```diff
@@ -0,0 +1,16 @@
+tasks = Table(
+    'tasks',
+    metadata,
+    Column('id', UUID, nullable=False),
```
server_default=sqlalchemy.text('gen_random_uuid()')

mestolin commented on db/tables/users.py:
```diff
@@ -0,0 +1,13 @@
+users = Table(
+    'users',
+    metadata,
+    Column('id', UUID, nullable=False),
```
server_default=sqlalchemy.text('gen_random_uuid()')

Commit:
added UserRole class to user.py
added UUID, role column, changed some columns nullable=True in users.py
added column assignee, UUID, changed some columns nullable=True, added ForeignKey to columns 'created_by' and 'assignee'
